lib/executor.js

```diff
@@ -248,6 +248,12 @@ function defineModels(app, instructions) {
         if (typeof code === 'function') {
           debug('Customizing model %s', name);
           code(model);
+        // If code is a default UMD module use it
+        } else if (code.hasOwnProperty('default')) {
```

Comment:
Can we make this check more robust by also checking for the presence of `code. __esModule`?